### ISSUE TITLE
Add nightmode action for Sonos Beam 

### ISSUE THREAD

**Issue by njudson11** [jishi/node-sonos-http-api#844]
Based on the documentation and the tests I've run the nightmode action is only available to PLAYBAR. Would it be possible to extend this feature so it also works on Sonos Beam devices as they support the night mode feature through the sonos app?

**Comment by jishi** [comment]
I have the beam and setting night mode works fine for me. Could you please
provide some logging from when you try to invoke the command?

**Comment by njudson11** [comment]
Yeah no problem. Although from what I can tell triggering the action doesn't seem to generate any extra logging. 

**Instruction:**
`http://*******:5005/Sonos%20Beam/nightmode/off`

**Response:**

```
{
  "error": "action 'nightmode' not found"
}
```

Log export attached:
[sonos-node-api.csv](https://github.com/jishi/node-sonos-http-api/files/9077911/sonos-node-api.csv)
